### ISSUE TITLE
pglogical_origin is mandatory for PostgreSQL 9.4

### ISSUE THREAD

**Issue by simaofreitas** [2ndQuadrant/pglogical#33]
Hi,

When trying to create the pglogical extension in a 9.4 server (producer), I get this error:
```
os=# CREATE EXTENSION pglogical;
ERROR:  required extension "pglogical_origin" is not installed
```

As per the README ( https://github.com/2ndQuadrant/pglogical#quick-setup ), it should only be necessary when having a 9.4 subscriber as well, but I'm going to subscribe from a 9.5. **note** this information is not available in the website ( https://2ndquadrant.com/en/resources/pglogical/pglogical-docs/ ).

Is this a copy error or the `pglogical_origin` should not be required for 9.4 in producer nodes?

Thanks

**Comment by ringerc** [comment]
PostgreSQL's extension dependencies mechanism has no way to specify that,
since it doesn't know what node is a producer or subscriber.

We could probably remove the extension dependency and check in pglogical
its self only when creating a subscriber. But then we lose the ability to
stop the user from dropping the extension unless we start fiddling with
catalog dependencies manually.

Having the extension installed is harmless. Why not just install it
unconditionally?

**Comment by simaofreitas** [comment]
@ringerc yes I agree. Then I guess I could make an improvement to the README to change:
```
If using PostgreSQL 9.4 as a subscriber, then the pglogical_origin extension also has to be installed on that node:
```
to 
```
If using PostgreSQL 9.4, then the pglogical_origin extension also has to be installed on that node:
```
I'll make a small pull request.

**Comment by ringerc** [comment]
Resolved by pull #34, thankyou very much.
